docs/Tutorials/zot-registry-tutorial.mdx

```diff
@@ -11,7 +11,7 @@ OCI registries are like a single location where container images and other artif
 
 We will be using zot registry in this guide. Zot registry is an OCI-native container registry for distributing container images and OCI artifacts.
 
-In order to follow the steps given, you would be required to install the ORAS CLI. You can follow the [installation guide](https://oras.land/docs/CLI/installation) to do so. 
+In order to follow the steps given, you would be required to install the ORAS CLI. You can follow the [installation guide](/docs/installation) to do so. 
```

Comment:
Looks like we need to first merge  - https://github.com/oras-project/oras-www/pull/216 as per @TerryHowe

Reply:
merged

Reply:
@TerryHowe @sajayantony all the links work now. Sorry for the inconvenience!